docs/checkit.js

```diff
@@ -354,12 +356,92 @@ CheckIt.prototype.onAuthStateChanged = function(user) {
     }
 };
 
-// Returns true if user is signed-in. Otherwise false and displays a message.
-CheckIt.prototype.checkSignedInWithMessage = function() {
-    // Return true if the user is signed in Firebase
-    if (this.auth.currentUser) {
-        return true;
-    }
+CheckIt.prototype.generateEmptyCalendar = function(calObj) {
+    // Generate the html for an empty calendar of the calendar you want to 
+    // display.
+    
+    var calendarDiv = this.$calendarDiv;
```

Comment:
The point of prefixing variables with `$` is so you know whether or not it's already jquery selector.